whylog_vim/output_formater/output_aggregator.py

```diff
@@ -20,11 +20,17 @@ def create_button(self, callback_function, line=None):
         line = line or self._get_current_line_number()
         self.buttons[line] = callback_function
 
+    def set_default_callback_function(self, default_callback_function):
+        self.default_callback_function = default_callback_function
+
     def call_button(self, line_number):
         try:
             self.buttons[line_number]()
         except KeyError:
-            pass
+            try:
+                self.default_callback_function()
```

Comment:
I didn't find definition of default_callback_function. I guess that will be implemented soon. It explain pass in except block. In the future mock unimplemented method